drupal-8/README.md

```diff
@@ -0,0 +1,155 @@
+# Working with Drupal8 in cloud.gov (and Cloud Foundry)
+
+[Drupal](https://drupal.org) has been a popular PHP-based framework for content-management systems and web applications. It was not written as a cloud-native framework, so it takes a bit of tweaking to run on cloud.gov, or any other Cloud Foundry implementation. If you're on a _greenfield_ project you may want to consider other PHP frameworks, but if you're going forward with Drupal then this guide will get you there.
+
+This guide is written for [cloud.gov](https://cloud.gov/) users, but will work for any Cloud Foundry site. Just replace the specifics for `aws-rds` and `s3` with your site equivalents and everything should just work. 
+
+## Quickstart
+
+You can demonstrate a fully-functional Drupal install assuming you've already cloned this repo, and are using this directory:
+
+Update `manifest.yml` with:
+
+1. the correct admin name, `ACCOUNT_NAME`
+1. delete or change the `AWS_S3_ENDPOINT` if you're _not_ on cloud.gov
+
+Log in to Cloud Foundry (e.g. `cf login --sso -a https://api.fr.cloud.gov`), then run the following commands for cloud.gov:
+
+```
+cf create-service s3 basic-public d8ex-s3
+cf create-service aws-rds shared-mysql d8ex-db
+cf push d8ex --no-start -b  https://github.com/cloudfoundry/apt-buildpack.git
```

Comment:
Why do you push with `--no-start` and the apt buildpack, and then push with the php buildpack separately? Why not:
# Push the app with `--no-start` and both buildpacks
# Set the password env var
# `cf start`

Reply:
@mogul I'm following the Cloud Foundry docs at https://docs.cloudfoundry.org/buildpacks/use-multiple-buildpacks.html where it states,

> Note: The two-push workflow is currently required because only v3-push supports multiple buildpacks.

Should I explain that here?